README.md

```diff
@@ -1,13 +1,38 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
-
-## Available Scripts
-
-In the project directory, you can run:
-
-### `npm start`
-
-Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+# Horondi
+Frontend app for the `Hondri` online store
+## Rules and guidelines
+- Redux
+    - For each entity we should have separate folder
+    - In each folder we should have different files for actions, reducer
+      `{modelName}.actions.js` or `{modelName}.reducer.js`
+- Configuration
+    - Configuration is done via `.env` file where environment
+      variables are located
+    - Also we have `.env.example` that contains examples of environment
+      variables
+- Styles
+    - For styling function `makeStyles` from `@material-ui`
+      should be used and all styles should be located inside separate
+      component.
```

Comment:
Consider changing "all styles should be located inside separate component" to  "all styles should be located inside a separate file `{modelName}-styles.js` in the component folder.